infra/src/main/kotlin/fi/fta/geoviite/infra/ratko/model/RatkoCommonModel.kt

```diff
@@ -171,8 +173,32 @@ enum class RatkoAccuracyType(@get:JsonValue val value: String) {
     @Suppress("unused") ESTIMATED_TRACK_ADDRESS("ESTIMATED TRACKADDRESS"),
 }
 
+data class RatkoPlan(
+    val id: Int?,
```

Comment:
Tämä pisti miettimään että saisiko näitä jotenkin fiksusti muutettua primitiiveistä vaikka IntId:ksi tms, jotta kentän luonne olisi tyypissä niinkuin muussakin geoviite koodissa. Mutta nämä taitaa olla kaikki suoraan ratkolta tulevien json-olioiden mallinnuksia, joten se ei ehkä oikein helpolla onnistu.